crates/wapc-codec/src/lib.rs

```diff
@@ -44,7 +44,6 @@
   clippy::equatable_if_let,
   bad_style,
   clashing_extern_declarations,
-  const_err,
```

Comment:
@flavio @jsoverson Does removing this require the `wapc-codec`, `wapc-guest`, `wapc-pool`, `wasm3-provider`, and `wapc` crates to be version bumped in addition to the `wasmtime-provider`?